pkgs/utils/src/log.zig

```diff
@@ -104,21 +123,26 @@ pub const DEFAULT_FILE_BEHAVIOUR = FileBehaviourParams{
     .fileActiveLevel = .debug,
     .filePath = "./log",
     .fileName = "consensus",
+    .monocolorFile = false,
 };
 
 pub const ZeamLogger = struct {
     activeLevel: std.log.Level,
     scope: LoggerScope,
     fileParams: ?FileParams,
+    testLogger: bool,
 
     const Self = @This();
-    pub fn init(scope: LoggerScope, activeLevel: std.log.Level, fileBehaviourParams: ?FileBehaviourParams) Self {
+    pub fn init(scope: LoggerScope, activeLevel: std.log.Level, fileBehaviourParams: ?FileBehaviourParams, testLogger: bool) Self {
```

Comment:
why is this extra flag testLogger needed, simply null should be passed as fileBehaviourParams in that case , which is just what getTestLogger would do